### DISCUSSION TITLE
biosphere/%D0%A2%D0%B5%D0%BE%D1%80%D0%B8%D1%8F/%D0%92%D0%B2%D0%B5%D0%B4%D0%B5%D0%BD%D0%B8%D0%B5

### DISCUSSION THREAD

**Discussion by giscus** [askurashev/ASKEducationPublic#4] [General]
# biosphere/%D0%A2%D0%B5%D0%BE%D1%80%D0%B8%D1%8F/%D0%92%D0%B2%D0%B5%D0%B4%D0%B5%D0%BD%D0%B8%D0%B5

Архив курса «Учение о биосфере и глобальные экологические проблемы»

https://education.kurashev.com/biosphere/%D0%A2%D0%B5%D0%BE%D1%80%D0%B8%D1%8F/%D0%92%D0%B2%D0%B5%D0%B4%D0%B5%D0%BD%D0%B8%D0%B5.html

**Comment by askurashev** [comment]
Good!
